src/context.tsx

```diff
@@ -16,23 +25,23 @@ export function getEmptyFilters(): Filters {
     isotypes_reported: new Set(),
     specimen_type: new Set(),
     publish_date: new Set([new Date(2019, 1, 1, 1), new Date()]),
-    estimate_grade: new Set()
-  }
+    estimate_grade: new Set(),
+  };
 }
 
 export function getDefaultFilters(): Filters {
```

Comment:
might need to double check but I don't think we need getDefaultFilters anymore as this was for the analyze tab

Reply:
Great catch!